packages/react/src/schematics/application/application.ts

```diff
@@ -1,63 +1,27 @@
-import { join, JsonObject, normalize, Path } from '@angular-devkit/core';
 import {
-  apply,
   chain,
-  externalSchematic,
-  filter,
-  mergeWith,
-  move,
-  noop,
   Rule,
   SchematicContext,
-  template,
   Tree,
-  url,
 } from '@angular-devkit/schematics';
-import {
-  addLintFiles,
-  formatFiles,
-  generateProjectLint,
-  insert,
-  names,
-  NxJson,
-  offsetFromRoot,
-  toFileName,
-  updateJsonInTree,
-  updateWorkspace,
-} from '@nrwl/workspace';
-import {
-  addDepsToPackageJson,
-  appsDir,
-  updateWorkspaceInTree,
-} from '@nrwl/workspace/src/utils/ast-utils';
-import { toJS } from '@nrwl/workspace/src/utils/rules/to-js';
-import * as ts from 'typescript';
-import { assertValidStyle } from '../../utils/assertion';
-import { addInitialRoutes } from '../../utils/ast-utils';
-import { updateJestConfigContent } from '../../utils/jest-utils';
+import { addLintFiles, formatFiles } from '@nrwl/workspace';
 import { extraEslintDependencies, reactEslintJson } from '../../utils/lint';
-import { CSS_IN_JS_DEPENDENCIES } from '../../utils/styled';
-import {
-  reactRouterDomVersion,
-  typesReactRouterDomVersion,
-} from '../../utils/versions';
 import init from '../init/init';
 import { Schema } from './schema';
-
-interface NormalizedSchema extends Schema {
-  projectName: string;
-  appProjectRoot: Path;
-  e2eProjectName: string;
-  parsedTags: string[];
-  fileName: string;
-  styledModule: null | string;
-  hasStyles: boolean;
-}
+import { createApplicationFiles } from './lib/create-application-files';
```

Comment:
Just moved these private helpers to their own files. Their content didn't change, except `addStyledModuleDependencies` now also adds the babel plugin as dependencies since we no longer install them with `@nrwl/react`.

Reply:
While you're at it, you might as well move this to `packages/react/src/utils/rules` or `pacakges/react/src/rules` :)